### DISCUSSION TITLE
Merging two nets

### DISCUSSION THREAD

**Discussion by Garpurinn** [ansys/pyaedt#2919] [Q&A] [answered]
Hi all, 

I am not sure if my methodology/poor programming skills are the issue here or some error is the cause, but I hope someone can help me with this issue. 
I am using the EDB part (pyaedt.Edb) to prepare my circuit design before importing into SiWave for further work/analysis. I import the IPC2581 file using the "edb.import_layout_pcb" function and perform modification on the stackup and materials. That works without any hiccups.

I also want to find all the separation points in the design, remove them and then unite the two nets that were connected to the separation point. I have no issues finding all the separation points and deleting them, but I have not been able to successfully unite the two nets. 
I have tried using "edb.core_nets.merge_nets_polygons([nets[0], nets[1]])", but that results in most of one the net disappearing (leaving mostly VIAs behind) and no merging of the nets.

Are there some better ways to achieve my goals or some issues with the "edb.core_nets.merge_nets_polygons" function? 

In advanced, thanks! 
Bjarki

**Comment by svandenb-dev** [comment, marked as answer]
Hi @Garpurinn,
this command merge_nets_polygons will merge 2 nets, and is expecting to get a list on net name (e.g. str). If this is not working as expected please submit an issue on Github. 
thanks for your feedback.
